.github/workflows/samples-java-helidon.yaml

```diff
@@ -18,10 +18,10 @@ jobs:
       matrix:
         sample:
           - samples/client/petstore/java-helidon-client/mp
-          - samples/client/petstore/java-helidon-client/se
```

Comment:
I assume this is temporary until a follow-on PR adds Helidon 4 SE support.

Reply:
I'm actually in the process of reworking the samples stuff a bit. We want both Helidon 3 and Helidon 4 samples which require different configs to drive the generator to produce the right output. Same for testing the generators.

Stay tuned.